docs/en/tutorials/getting-started.rst

```diff
@@ -616,12 +613,13 @@ domain model to match the requirements:
         }
     }
 
```

Comment:
This section regarding ArrayCollection seemed to be incorrect, since no objects of type "Doctrine" are involved here.  Please confirm the accuracy of the re-write, though.